OpenRA.Mods.Cnc/Traits/SupportPowers/IonCannonPower.cs

```diff
@@ -27,8 +27,12 @@ class IonCannonPowerInfo : SupportPowerInfo, IRulesetLoaded
 		[Desc("Amount of time to keep the camera alive")]
 		public readonly int CameraRemoveDelay = 25;
 
-		[Desc("Effect sequence to display")]
+		[Desc("Effect sequence sprite collection")]
```

Comment:
Please change the description to "Effect image" to match everything else.  There's no need to introduce a new term for them.